File: src/test/java/com/xiang/StudentTest.java
package com.xiang;

import com.tom.student.Student;
import org.junit.jupiter.api.Assertions;
import org.junit.jupiter.api.Test;

public class StudentTest {

    @Test
    public void highestTest() {
        Student student = new Student("xiang",70,100);
        Assertions.assertEquals(100,student.highest());
    }
    @Test
    public void averageTest() {
        Student student = new Student("xiang",80,90);
        Assertions.assertEquals((80+90)/2,student.getAverage());
    }
}

File: src/test/kotlin/com/xiang/StudentTest.kt
package com.xiang.kotlin

import com.tom.kotlin.student.student
import org.junit.jupiter.api.Assertions
import org.junit.jupiter.api.Test

class StudentTest {

    @Test
    fun highestTest() {
        val student = student("xiang",80,100)
        Assertions.assertEquals(100,student.highest())
    }

    @Test
    fun averageTest() {
        val student = student("xiang",80,100)
        Assertions.assertEquals((80+100)/2,student.getAverage())
    }

    @Test
    fun gradingTest() {
        val student = student("xiang",80,100)
        Assertions.assertEquals('A',student.grading())
    }
}